Stellar-ledger.x

```diff
@@ -490,11 +490,46 @@ struct LedgerCloseMetaV1
     SCPHistoryEntry scpInfo<>;
 };
 
+struct LedgerCloseMetaV2
+{
+    LedgerHeaderHistoryEntry ledgerHeader;
+
+    GeneralizedTransactionSet txSet;
+
+    // NB: transactions are sorted in apply order here
+    // fees for all transactions are processed first
+    // followed by applying transactions
+    TransactionResultMeta txProcessing<>;
+
+    // upgrades are applied last
+    UpgradeEntryMeta upgradesProcessing<>;
+
+    // other misc information attached to the ledger close
+    SCPHistoryEntry scpInfo<>;
+
+    // Size in bytes of BucketList, to support downstream
+    // systems calculating storage fees correctly.
+    uint64 totalByteSizeOfBucketList;
+
+    // Expired temp keys that are being evicted at this ledger.
+    LedgerKey evictedTemporaryLedgerKeys<>;
+
+    // Expired restorable ledger entries that are being
+    // evicted at this ledger.
+    LedgerEntry evictedRestorableLedgerEntries<>;
+
+    // We forgot to add an ExtensionPoint in v1 but at least
```

Comment:
Should the extension point be at the top like [this](https://github.com/stellar/stellar-xdr/blob/07e9e82b8d60244596214b4d327506d791d35ad3/Stellar-ledger-entries.x#L139-L141) for more flexibility (the comment explains why).

Reply:
Oh, sure. Moved to the top of the struct.